03-task-manager/starter/routes/tasks.js

```diff
@@ -0,0 +1,15 @@
+const express = require("express");
+const router = express.Router();
+
+const {
+	getAllTasks,
+	createTask,
+	getTask,
+	updateTask,
+	deleteTask,
+} = require("../controllers/tasks");
+
+router.route("/").get(getAllTasks).post(createTask);
+router.route("/:id").get(getTask).patch(updateTask).delete(deleteTask);
```

Comment:
How is this possible in JS to have chained code like above?  The design pattern is called Builder Design Pattern, which is used very much in Node. Learn more about it, if you get time.